spec/03-plugins/21-ldap-auth/02-invalidations_spec.lua

```diff
@@ -28,29 +27,36 @@ for _, strategy in helpers.each_strategy() do
           ldap_port = "389",
           start_tls = false,
           base_dn   = "ou=scientists,dc=ldap,dc=mashape,dc=com",
-          attribute = "uid"
+          attribute = "uid",
+          cache_ttl = 5,
```

Comment:
Could we use a slightly shorter TTL, to avoid slowing down our tests? 5s isn't much, but when every single test relying on TTL behaviour in the suite waits for 5s between each test, we end up with a problem... Maybe 1? 2?

Reply:
sure, will update it.

Reply:
I set it to one, please review